bane-of-wargs.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<link rel="stylesheet" type="text/css" href="style.css">
+	<title>Bane Of Wargs</title>
+	<body>
+	  <div class="header">
+	    <a href="index.html">Home</a> | <a href="running.html">Running Dungeons of Kathallion</a> | <a href="development.html">Development</a> | <span class="selected">Bane Of Wargs</span>
+	  </div>
+	  <div class="inner">
+
+    <img src="bane of wargs.png" width="620" />
+      
+    <p>What is BANE OF WARGS?</p>
+	<p>Bane Of Wargs is at first a game engine, that you can use to create any story, item, mount, weapon, villages or anything you'd imagine. It is free, and open-source, you can fork the game engine and tweak as you would like, to make it like you'd like!</p>
+	<p>You have the ability to create mods/plugins, that you can load in the main menu and the mod content will overwrite the vanilla content, which make you able to totally modify the story of the base game.</p>
+	<p>You can find the game repository at <a href="https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs" target="_blank">https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs</a></p>
+	<p></p>
+	<p>Running The Game:</p>
+	<p class="dialog">git clone git@github.com:Dungeons-of-Kathallion/Bane-Of-Wargs.git</p>
+	<p class="dialog">cd Bane-Of-Wargs/</p>
+	<p class="dialog">pip install -r requirements.txt</p>
+	<p class="dialog">chmod +x main.py battle.py colors.py</p>
```

Comment:
What's this doing?

Reply:
Just this:
![image](https://github.com/Dungeons-of-Kathallion/Dungeons-of-Kathallion.github.io/assets/87318892/4af05948-d2e6-4887-a27d-f36a456c66ca)

Reply:
I mean what does the `chmod +x main.py battle.py colors.py` mean?

Reply:
It makes the scripts executable. It's basic bash command, you didn't knew of it?

Reply:
OK, sorry I don't know that much command line stuff so I'm not entirely following, but I might understand.